yaml_overview.rst

```diff
@@ -22,55 +65,96 @@ publish and navigate keys**
     - divider:
         do:
           divide:
-            - dividend: ${input1}
```

Comment:
I think we should leave these as originally written since our content repository always indents the lists.